medicines/doc-index-updater/README.md

```diff
@@ -40,9 +40,9 @@ stunnel stunnel.conf
 make
 ```
 
-To get SFTP working locally, first set up a local user, with username and password, that you can use as a local SFTP server.
+To get SFTP working locally, first set up a local user, with a username that you can use as a local SFTP server (e.g. localsftpuser).
```

Comment:
I think you still need to set a password which is used if you want to manually log in to the sftp server?

Reply:
you do actually need the password too when you run the line that adds it to authorised_keys